src/libertem/io/dataset/dm.py

```diff
@@ -230,6 +258,10 @@ def initialize(self, executor):
         first_file = next(self._fileset.files_from(0))
         nav_dims = self._get_scan_size()
         shape = nav_dims + tuple(first_file.shape)
+        newShape = list()
+        for values in shape:
+            newShape.append(int(values))
+        shape = tuple(newShape)
```

Comment:
Nitpick: this can be expressed more succinctly with [a list comprehension](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions), mapping `int` over `shape`.

Reply:
I added scan_size parameter but still it is showing NaN. I am trying to figure it out.

Reply:
[
![Screenshot from 2020-04-16 17-34-06](https://user-images.githubusercontent.com/24645773/79454310-9c624100-8008-11ea-924a-1bce26e365f7.png)
](url)
After updating the scan_size and running FFT analysis I am getting black screen as result of analysis.